mitiq/interface/conversions.py

```diff
@@ -81,6 +81,11 @@ def convert_to_mitiq(circuit: QPROGRAM) -> tuple[cirq.Circuit, str]:
     """
     conversion_function: Callable[[Any], cirq.Circuit]
 
+    if isinstance(circuit, str) and hasattr(circuit, "__module__") is False:
```

Comment:
```suggestion
    if isinstance(circuit, str):
```

WDYT of simplifying this `if`? Technically what you have is more correct, but I think it's a bit more readable and not much of a safety sacrifice to remove it.

Reply:
Ahh okay good find. Thanks for the explanation.